src/rest/api_call.cpp

```diff
@@ -43,7 +45,8 @@ void rest_client::perform_request(std::string uri, rest_request_type method, htt
         if(response.code == 429) {  // Being rate limited
             int wait_for = message["retry_after"].get<int>();
             debug::log(debug::log_level::INFORMATIONAL, debug::log_origin::REST, "API request exceeds rate limts. Retrying in " + std::to_string(wait_for) + "ms");
-            boost::this_thread::sleep(boost::posix_time::milliseconds(wait_for));
+            // This is kinda confusing but we have to specify when to wait.
```

Comment:
oh haha this is a comment from when I had code that was 5 times more complex than this. I just forgot to remove it, I'll remove it with a force-push.